packages/next/test/fixtures/36-instrumentation-hook/probes.json

```diff
@@ -0,0 +1,14 @@
+{
+  "probes": [
+    {
+      "path": "/",
+      "status": 200,
+      "mustContain": "false"
+    },
+    {
+      "path": "/api/hello",
+      "status": 200,
+      "mustContain": "true"
```

Comment:
Can we be more specific for these true/false assertions? My worry is that it could match any part of the payload.